doc/library/tensor/random/basic.rst

```diff
@@ -47,7 +47,7 @@ Reference
 Distributions
 ==============
 
-Aesara can produce :class:`RandomVariable`\s that draw samples from many different statistical distributions, using the following :class:`Op`\s.
+Aesara can produce :class:`RandomVariable`\s that draw samples from many different statistical distributions, using the following :class:`Op`\s. :class:`RandomVariable`\s are what NumPy calls *Generalized Universal Functions*: each :class:`Op` carries information that states what the "core" dimensionality of the input is, as well as the corresponding dimensionality of the outputs. The list of core dimensions for each parameter of the random variable is called its *signature*.
```

Comment:
This is a bit misleading, because `RandomVariable`s aren't technically `gufunc`s in Aesara (or NumPy?)